cc.c

```diff
@@ -275,8 +275,7 @@ int main(int argc, char** argv)
 
 	if(Architecture == KNIGHT_NATIVE
 		|| Architecture == KNIGHT_POSIX
-		|| Architecture == ARMV7L
-		|| Architecture == AARCH64)
+		|| Architecture == ARMV7L)
```

Comment:
I suspect this one needs to go the other direction too...

Reply:
Just checked and I believe so as well. It also seems I made the same mistake for reserving locals where I subbed from the the immediate value instead of from the stack value. I'll put a PR up.